langchain4j-vertex-ai-gemini/src/main/java/dev/langchain4j/model/vertexai/VertexAiGeminiChatModel.java

```diff
@@ -293,6 +301,18 @@ public Response<AiMessage> generate(List<ChatMessage> messages, ToolSpecificatio
         }
     }
 
+    @Override
+    public Set<Capability> supportedCapabilities() {
+        Set<Capability> capabilities = new HashSet<>();
+        // When response format is not null, it's always application/json or text/x.enum as mime type.
+        // GenerationConfig defaults to an empty schema when a response schema is not specified,
+        // so we check that the response format schema is equal to a default (empty) schema.
+        if (Schema.getDefaultInstance().equals(this.generationConfig.getResponseSchema())) {
```

Comment:
Should it be "not equals" then?

But maybe we should check `"application/json".equals(responseMimeType)` instead?

`RESPONSE_FORMAT_JSON_SCHEMA` means "this model can accept `JsonSchema`", not "user configured specific `JsonSchema` when creating the model"